src/main/java/util/LocalLoaderImpl.java

```diff
@@ -25,6 +25,14 @@ public String getBasePathUrl() {
         return properties.getProperty("base.path.url");
     }
 
+    public String getApiKeys() {
```

Comment:
Should we also update the readme instructions for that? It seems that we are adding more properties in our config file ?

Reply:
Readme file is update with the required config change